sdk/formrecognizer/Azure.AI.FormRecognizer/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@
 ### Features Added
 
 ### Breaking Changes
+- Renamed method `StartCreateComposedModel` to `StartComposeModel`.
+- Renamed `BuildModelOptions.ModelDescription` to `Description`.
+- Renamed `modelDescription` parameters to `description` in methods `GetCopyAuthorization` and `StartCreateComposedModel` (now called `StartComposeModel`).
+- Renamed `CopyAuthorization.ExpirationDateTime` to `ExpiresOn`.
```

Comment:
Curious why some of these were not done in the swagger itself - so every language could benefit from it?

Reply:
Ahh looks like they were done in swagger only. We had a convenience layer added on top so had to manually change it as well I guess

Reply:
- `StartCreateComposedModel`: this is a method in the client, so there's no corresponding change to do in the swagger. All other languages are making the same change.
- `ModelDescription`: for some reason .NET was the odd one out here. The swagger and other languages had it as `description` only.
- `ExpirationDateTime`: changing it to `ExpiresOn` is .NET-specific (we usually have dates named like `CreatedOn`, `LastUpdatedOn`, etc).